src/main/java/cn/nukkit/utils/BinaryStream.java

```diff
@@ -520,21 +557,72 @@ public void putSlot(int protocolId, Item item) {
         }
 
         int networkId = item.getId();
+
+        // Multiversion: Replace unsupported items
```

Comment:
I moved the location of this code so that when 1.17 is updated, there is no need to move it again (Google Translate)